src/server/models/relationship.model.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+let relationshipSchema = new Schema({
+    mentorId: String,
+    menteeId: String,
+    check: Boolean,
+    matchDate: {
```

Comment:
What's a match date? Is it the requested date, or the accepted date? Either support both, or remove both.